community/sway/etc/skel/.config/waybar/config

```diff
@@ -112,12 +112,6 @@
         "max-length": 120
     },
 
-    "sway/workspaces": {
-        "all-outputs": false,
-        "disable-scroll": true,
```

Comment:
this was the only non-default and I don't mind / think users can decide themselves if they find it annoying.